tests/test_observatory_envar.py

```diff
@@ -0,0 +1,84 @@
+import os
+import sys
+import pytest
+import importlib
+
+import pint.observatory.observatories
+import pint.observatory.special_locations
+import pint.observatory
+
+
+@pytest.fixture
+def sandbox(tmp_path):
```

Comment:
Why is this in a separate file? What is gained? Imports are still global across python invocations so this doesn't separate the registry or the imports from the non-envar version of this file.

Reply:
It was just for simplicity when I was testing.  if that violates some principle I can move it back